src/AcceptanceTests/Routing/When_unsubscribing_from_one_of_the_events_for_ForwardingTopology.cs

```diff
@@ -0,0 +1,138 @@
+﻿namespace NServiceBus.Azure.Transports.WindowsAzureServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Linq;
+    using System.Security.Cryptography;
+    using System.Text;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using Features;
+    using Microsoft.ServiceBus;
+    using NUnit.Framework;
+    using Settings;
+    using Transport.AzureServiceBus;
+    using AzureServiceBus;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+
+    public class When_unsubscribing_from_one_of_the_events_for_ForwardingTopology : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_remove_subscription_rule_from_topology()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Endpoint>(b => b.When((session, c) => session.Unsubscribe<MyEvent>()))
+                .Done(c => c.EndpointsStarted)
+                .Run();
+
+            var connectionString = Environment.GetEnvironmentVariable("AzureServiceBusTransport.ConnectionString");
+            var namespaceManager = NamespaceManager.CreateFromConnectionString(connectionString);
+            var rawEndpointName = ConfigureEndpointAzureServiceBusTransport.NameForEndpoint<Endpoint>();
+            var endpointName = MD5HashBuilder.Build(rawEndpointName);
+            var ruleName = MD5HashBuilder.Build(typeof(MyEvent).FullName);
+
+            if (context.IsForwardingTopology)
+            {
+                var isSubscriptionFound = await namespaceManager.SubscriptionExistsAsync("bundle-1", endpointName);
+                Assert.IsTrue(isSubscriptionFound, "Subscription under 'bundle-1' should have been found, but it wasn't.");
+
+                isSubscriptionFound = await namespaceManager.SubscriptionExistsAsync("bundle-2", endpointName);
+                Assert.IsTrue(isSubscriptionFound, "Subscription under 'bundle-2' should have been found, but it wasn't.");
+
+                var rules = await namespaceManager.GetRulesAsync("bundle-1", endpointName);
+                Assert.That(rules.Any(rule => rule.Name == ruleName), Is.False);
```

Comment:
Disagree. 
We've got another tests that verifies that a single event subscription will cause removal of the subscription and rule. In this scenario, we only care about specific event subscription. If there are other events (which we shouldn't care about) , then subscription won't be removed. In case subscription is not found (tested here), then we'll fail as should have.
We care for the specific event to be removed and for the subscription to remain if there are other events. We've got another test that verifies